src/components/ebay-table/table.stories.ts

```diff
@@ -1,8 +1,11 @@
+/// <reference types="vite/client" />
```

Comment:
@agliga lmk if you don't like this line, it removes error prompt for importing raw marko file.

Reply:
Lets remove it. There's another pr that resolves that i thought